spec/models/task_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: tasks
+#
+#  id            :uuid             not null, primary key
+#  description   :text
+#  position      :integer
+#  to_do_list_id :bigint
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#
+RSpec.describe Task, type: :model do
+  it 'has correct parrent' do
+    expect(subject).to be_a_kind_of(ApplicationRecord)
+  end
+
+  describe 'associations' do
+    it { should belong_to(:to_do_list).class_name('ToDoList') }
```

Comment:
please, don't use should syntax -it's 4 year as it's outdated.

Use it { is_expected.... }